docs/markdown/i18n-module.md

```diff
@@ -74,3 +74,72 @@ for normal keywords. In addition it accepts these keywords:
 * `mo_targets` *required*: mo file generation targets as returned by `i18n.gettext()`.
 
 *Added 0.62.0*
+
+
+### i18n.pot_extractor()
+
+``` meson
+i18n.pot_extractor(args: [...], required: true)
+```
+
+Find the `xgettext` program, and return a `PotExtractor` object.
+`args` are a list of arguments to provide to `xgettext`.
+If `required` is `true` and `xgettext` cannot be found, it will result in
+an error.
+
+This function is to be used when the `gettext` function workflow it not suitable
+for your project. For instance, our project is a monorepo with several libraries
+and some executables depending of those libraries. We need to extract translations
+from each library, but for each executable, we want to generate a pot file with
+only the translations from the linked libraries (transitively). 
```

Comment:
Do not mention "your project".

```suggestion
for your project. For example, it can be used to produce separate `.pot` files for each executable.
```